HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -2925,6 +3058,26 @@ def self.apply_hvac_sizing(hpxml, weather, cfa)
           htg_sys.additional_properties.GSHP_Bore_Depth = hvac_sizing_values.GSHP_Bore_Depth
           htg_sys.additional_properties.GSHP_Bore_Holes = hvac_sizing_values.GSHP_Bore_Holes
           htg_sys.additional_properties.GSHP_G_Functions = hvac_sizing_values.GSHP_G_Functions
+
+          geothermal_loop = htg_sys.geothermal_loop
+          if not geothermal_loop.nil?
+            if geothermal_loop.loop_flow.nil?
+              geothermal_loop.loop_flow = hvac_sizing_values.GSHP_Loop_flow
+              geothermal_loop.loop_flow_isdefaulted = true
+            end
+            if geothermal_loop.num_bore_holes.nil?
+              geothermal_loop.num_bore_holes = hvac_sizing_values.GSHP_Bore_Holes
+              geothermal_loop.num_bore_holes_isdefaulted = true
+            end
+            if geothermal_loop.bore_length.nil?
+              geothermal_loop.bore_length = hvac_sizing_values.GSHP_Bore_Depth # this is the length (i.e., depth) of each borehole?
```

Comment:
Looking at the code in hvac_sizing.rb, it sure seems like it is per borehole. Delete comment if you agree.